cordlib/src/test/kotlin/com/luxoft/blockchainlab/corda/hyperledger/indy/ReadmeExampleTest.kt

```diff
@@ -87,20 +88,22 @@ class ReadmeExampleTest : CordaTestBase() {
 
         // Alice.BORN >= currentYear - 18
         val eighteenYearsAgo = LocalDateTime.now().minusYears(18).year
-        val legalAgePredicate =
-            ProofPredicate(schemaId, credentialDefinitionId, "BORN", eighteenYearsAgo)
+        val youngerPredicate = ProofPredicate(schemaId, credentialDefinitionId, "BORN", eighteenYearsAgo)
 
-        val verified = store.services.startFlow(
+        val older = !store.services.startFlow(
```

Comment:
revert and ignore test and add todo with description. 
There is an issue because indy doesn't support required predicate.

Reply:
Maybe we should rearrange the test to complement the supported one?